src/codegen.cpp

```diff
@@ -9310,17 +9310,6 @@ extern "C" void jl_init_llvm(void)
     if (clopt && clopt->getNumOccurrences() == 0)
         cl::ProvidePositionalOption(clopt, "4", 1);
 
```

Comment:
```suggestion

    // we want the opaque-pointers to be opt-in, per LLVMContext, for this release
    // so change the default value back to pre-14.x, without changing the NumOccurrences flag for it
    clopt = llvmopts.lookup("opaque-pointers");
    if (clopt && clopt->getNumOccurrences() == 0) {
        clopt->addOccurrence(1, clopt->ArgStr, "false", true);
    }
```

Reply:
This isn't strictly required, as in https://github.com/maleadt/LLVM.jl/pull/371 I'll change LLVM.jl to explicitly disable opaque pointers by default, but I guess it can't hurt for compatibility with older versions of LLVM.jl.